spec/models/technology_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Technology, type: :model do
```

Comment:
Great work!!! just beware about the RSpec prefix since it breaks the styleguide: https://github.com/Wolox/tech-guides/blob/master/ruby-on-rails/docs/standards/rspec-style-guide.md#describe